examples/langgraph-sls-fastapi-rag/src/api/main.py

```
@@ -0,0 +1,20 @@
+from fastapi import FastAPI
```

Ruff (0.8.2)

Missing docstring in public module

(D100)